portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,26 +19,38 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.util.ArrayList;
+import java.util.Date;
+import com.google.gson.Gson;
+
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
     @Override
     public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-        response.setContentType("text/html;");
-        // empty line after the button
-        response.getWriter().println();
-        response.getWriter().println("Hello world!");
-        // The welcome message and show the data
-        response.getWriter().println("Hello, Jiaxi Chen!");
-        // Critical section, using sychronized to make sure mutual exclusion
-        synchronized(this) {
-            count++;
-            response.getWriter().println("You've visited this site " + count +
-                                            " times!");
-        }
+        String json = getInformationInJson();
+        response.setContentType("application/json");
+        response.getWriter().println(json);
     }
 
-    private int count = 0;
+    /**
+     * Creates an ArrayList containing three information, then converts it
+     * json.
+     * @return a String containing the json format of the those information
+     */
+    private static String getInformationInJson() {
+        // create an ArrayList containing three personal information
+        ArrayList<String> information = new ArrayList<>();
+        information.add("Jiaxi Chen");
+        information.add("Chongqing");
+
+        // get the current time and adds to the list
+        information.add(new Date().toString());
+
+        // convert it to json format and returns that json
+        String json = new Gson().toJson(information);
```

Comment:
again, you don't need to make a new variable.

A simple rule is: 
Does the variable name give any extra meaning?
```
String canonical_date = ...
```
Or is it complicated and worth documenting?
```
// ...
int magic_number = 0x1234;
```
Or is it too long to read (readers might not understand at the first glance)?
```
String canonical_json = Process(new Gson().toJson(...).Append(...));
```

Reply:
Emm Curious, why can you merge into the master without addressing this comment?
I submit this review with "Request change"....

Reply:
I don't know. But, it said you've approved this request. Probably you request change and then click the approve by accident. I'll fix this issue in the next PR along with the step 4 for this week's project!